src/Settings.cc

```diff
@@ -22,6 +22,7 @@ void ISSSettings::ReadSettings() {
 	n_array_row = config->GetValue( "NumberOfArrayRows", 4 );
 	n_array_pstrip = config->GetValue( "NumberOfArrayPstrips", 128 );
 	n_array_nstrip = config->GetValue( "NumberOfArrayNstrips", 11 );
+	n_array_side = config->GetValue( "NumberOfArraySides", 6 );
```

Comment:
The number of sides here means sides of the semiconductor, i.e. p-side and n-side... Therefore this should be fixed to 2 and not changed. However, it is never actually used in the code, so maybe we can just delete it eventually. It won't break anything at the moment anyway.

Reply:
Removed in commit d426544